modules/rostests/apitests/shell32/ShellExecCmdLine.cpp

```diff
@@ -415,6 +414,7 @@ static void DoEntry(const TEST_ENTRY *pEntry)
             {
                 bFound = TRUE;
                 SendMessage(hwnd, WM_CLOSE, 0, 0);
+                Sleep(RETRY_INTERVAL);
```

Comment:
What is this for?

Reply:
It's for waiting for closing the window.